pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -435,11 +442,13 @@ type ConsulSDConfig struct {
 	// +optional
 	Services []string `json:"services,omitempty"`
 	// An optional list of tags used to filter nodes for a given service. Services must contain all tags in the list.
-	//+listType:=atomic
+	// +kubebuilder:validation:MinItems=1
```

Comment:
If we add the validation here, we should do the same for Services IMHO.

Reply:
AFAIK, services also has the same `listType=atomic` validation. Are you referring to something else?
I feel these lists could benefit from validation against duplicates with `listType=map`. WDYT?

Reply:
IIRC I meant validation about the min length.